### PR TITLE
Create Profile Picture Component

### PR DESCRIPTION
**Pull request by MarceloDiefenbach**
# What was done? ✅
 Describe in a few words what have you done? 
 - Step 1: Create a folder of Profile Picture Component inside DesignSystem/Component folder
 - Step 2: Implement the component that show user image and name
 - Step 3: Develop TabBar with challenge list and profile tabs
 - Step 4: after develop TabBar, i do some changes to navigations work (challenge details screen and first screen of create  
 - Step 5: quick challenge. Now this screens open as sheet withou dismiss action.
 - Step 6: Develop back button on challenge details screen
 - Step 7: develop profile component
 - Step 8: devolop profile screen with button to delete account  (TODO: in viewmodel do logic to delete account
 - Step 9: fix color of background on challenge detail screen
 - Step 10: Develop alert to confirm account delete (delete account)
 - Step 11: develop alert of server error (delete account)

# Description 📝
## Project Info 📈
 - US **08**
 - Task 
   - ID: **15**
   - Title: **Develop User Profile Picture Component**
  
# Evidence 🕵️‍♀️
## **Screenshots/Videos/Figma** 📱
Screenshot of login screen

### CODE REVIEW

**Review by jgabrielbiazus — APPROVED** [review]

File: Fiero.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1004,7 +1036,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_ASSET_PATHS = "\"Fiero/Preview Content\"";
-				DEVELOPMENT_TEAM = GYMDP69744;
+				DEVELOPMENT_TEAM = 6525BU9D84;
```
Comment (MarinaFX) [inline Fiero.xcodeproj/project.pbxproj:1039]: mudar development team

File: Fiero.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1020,7 +1052,7 @@
 					"@executable_path/Frameworks",
 				);
 				MARKETING_VERSION = 1.0;
-				PRODUCT_BUNDLE_IDENTIFIER = com.pazzi.marina.FieroApp;
+				PRODUCT_BUNDLE_IDENTIFIER = com.pazzi.marina.FieroApp4;
```
Comment (MarinaFX) [inline Fiero.xcodeproj/project.pbxproj:1055]: mudar development team

File: Fiero.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -1036,7 +1068,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_ASSET_PATHS = "\"Fiero/Preview Content\"";
-				DEVELOPMENT_TEAM = GYMDP69744;
+				DEVELOPMENT_TEAM = 6525BU9D84;
```
Comment (MarinaFX) [inline Fiero.xcodeproj/project.pbxproj:1071]: mudar development team

File: Fiero.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -1052,7 +1084,7 @@
 					"@executable_path/Frameworks",
 				);
 				MARKETING_VERSION = 1.0;
-				PRODUCT_BUNDLE_IDENTIFIER = com.pazzi.marina.FieroApp;
+				PRODUCT_BUNDLE_IDENTIFIER = com.pazzi.marina.FieroApp4;
```
Comment (MarinaFX) [inline Fiero.xcodeproj/project.pbxproj:1087]: voltar package anterior

File: Fiero.xcodeproj/xcuserdata/marcelodiefenbach.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Bucket
+   uuid = "605D32B1-6DC0-49DE-9610-2D8107A17EC1"
+   type = "1"
+   version = "2.0">
+</Bucket>
```
Comment (MarinaFX) [inline Fiero.xcodeproj/xcuserdata/marcelodiefenbach.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist:6]: nao subir esse arquivo

File: Fiero/Views/Challenges/Quick/QCCategorySelectionView.swift [hunk 6]
```diff
@@ -11,48 +11,51 @@ struct QCCategorySelectionView: View {
...
     
     var body: some View {
-        VStack {
-            Text("Escolha um novo desafio rápido")
-            .multilineTextAlignment(.center)
-            .font(Tokens.FontStyle.largeTitle.font(weigth: .bold, design: .rounded))
-            .padding(.horizontal, Tokens.Spacing.xs.value)
-            .padding(.vertical, Tokens.Spacing.sm.value)
-            .foregroundColor(Tokens.Colors.Neutral.High.pure.value)
+        NavigationView{
```
Comment (MarinaFX) [inline Fiero/Views/Challenges/Quick/QCCategorySelectionView.swift:14]: talvez seria melhor tirar essa navigation view, senao vamos ter 2

**Review by MarinaFX — CHANGES_REQUESTED** [review]

**Pull request merged**